server/src/plugin/runtime/install-python.ts

```diff
@@ -0,0 +1,19 @@
+import { PortablePython } from "@bjia56/portable-python";
+import path from "path";
+import { getScryptedVolume } from "../plugin-volume";
+
+export const PYTHON_VERSION = "3.9";
+
+function pythonInstallDir() {
+    return path.join(getScryptedVolume(), "python");
+}
+
+export function getPythonExecutable() {
+    const portablePython = new PortablePython(PYTHON_VERSION, pythonInstallDir());
+    return portablePython.executablePath;
+}
+
+export async function ensurePythonInstall() {
+    const portablePython = new PortablePython(PYTHON_VERSION, pythonInstallDir());
+    await portablePython.install();
```

Comment:
Similar to how other native binaries are packaged (ffmpeg-static, level), the install of native binaries should be done at `npm install` time via a npm `postinstall` script.

Reply:
this script will need to be in the portable-python npm lib